applications/trilinos_application/external_includes/amgcl_mpi_schur_complement_solver.h

```diff
@@ -372,7 +372,9 @@ class AmgclMPISchurComplementSolver : public LinearSolver< TSparseSpaceType,
             mPressureMask.resize( rA.NumMyRows(), false );
 
         int counter = 0;
+#ifdef KRATOS_DEBUG
```

Comment:
Is it used in DEBUG?, maybe better remove it

Reply:
yes it is only used in debug
@RiccardoRossi should decide if it is to be removed

Reply:
this is only a debugging output. Also the solver is not yet fully tested. I would leave it for now